### PR TITLE
add kam to ci-team.members.txt

### PR DESCRIPTION
**Pull request by kkasravi**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/internal-acls/74)

### CODE REVIEW

File: ci-team.members.txt [hunk 1]
```diff
@@ -10,3 +10,4 @@ redhat-team@kubeflow.org
 wibuch@microsoft.com
 yangpa@google.com
 holden.karau@gmail.com
+kam.d.kasravi@intel.com
```
Comment (abhi-g) [inline ci-team.members.txt]: Kam, looks like most of the file was alphabetical in order except for holden. Can I request that you move your entry and holden's entry to appropriate order.

**Comment by jlewi** [comment]
ping @kkasravi

**Comment by kkasravi** [comment]
@abhi-g - should be sorted now

**Comment by abhi-g** [comment]
Thanks @kkasravi

**Pull request merged**